Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM debian:sid
+
+RUN apt-get update && apt-get install -y build-essential git ant default-jdk zip
+RUN git clone https://github.com/facebook/buck.git
```

Comment:
The cloning is not working for me.

Additionally in order to run git clone you must first add github to the list of known hosts with 

```
RUN mkdir -p /root/.ssh
# Add github key
RUN ssh-keyscan github.com >> /root/.ssh/known_hosts
RUN chmod 600 /root/.ssh/known_hosts
```